src/nft-card.ts

```diff
@@ -171,6 +172,7 @@ export class NftCard extends LitElement {
     // Get the web3 provider
     this.provider = getProvider()
 
+    this.network = networkFromString(this.network)
```

Comment:
instead of mutating the same variable, can we make a new one, e.g. networkName, to pass to the SDK?

And it doesn't need to be an instance var right? it can be a const?

Reply:
@alexanderatallah done, let me know if that was what you were thinking

Reply:
@alexanderatallah `this.network` get passed down to `<nft-card-front/>` in `renderInnerCardTemplate` so it will need to be an instance var AFAIK

Reply:
Ok. small point, I would rename all internal usages to avoid mutating variables but you should prioritize your other PRs